runtime/doc/lsp.txt

```diff
@@ -58,13 +58,16 @@ options are not restored when the LSP client is stopped or detached.
 - 'formatexpr' is set to |vim.lsp.formatexpr()|, so you can format lines via
   |gq| if the language server supports it.
   - To opt out of this use |gw| instead of gq, or set 'formatexpr' on LspAttach.
+- |K| is mapped to |vim.lsp.buf.hover()| unless |'keywordprg'| is customized or
+  a custom keymap for `K` exists.
```

Comment:
Also documented in vim_diff.txt and `:help K-lsp-default` : https://github.com/neovim/neovim/pull/24351